src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/RadioButtonRendererTests.cs

```diff
@@ -0,0 +1,175 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Drawing;
+using System.Windows.Forms.Metafiles;
+using System.Windows.Forms.VisualStyles;
+
+namespace System.Windows.Forms.Tests;
+
+public class RadioButtonRendererTests : AbstractButtonBaseTests
+{
+    [WinFormsTheory]
+    [InlineData(RadioButtonState.CheckedNormal)]
+    [InlineData(RadioButtonState.CheckedPressed)]
+    public void RadioButtonRenderer_DrawRadioButton(RadioButtonState state)
+    {
+        using Bitmap bitmap = new(100, 100);
+        using Graphics graphics = Graphics.FromImage(bitmap);
+        Point point = new Point(10, 20);
+
+        RadioButtonRenderer.DrawRadioButton(graphics, point, state);
+    }
+
+    [WinFormsTheory]
+    [InlineData(RadioButtonState.CheckedNormal)]
+    [InlineData(RadioButtonState.CheckedPressed)]
+    public void RadioButtonRenderer_DrawRadioButton_OverloadWithSizeAndText(RadioButtonState state)
+    {
+        using Bitmap bitmap = new(100, 100);
+        using Graphics graphics = Graphics.FromImage(bitmap);
+        Point point = new Point(10, 20);
+        Rectangle bounds = new Rectangle(10, 20, 30, 40);
```

Comment:
```suggestion
        Rectangle bounds = new(10, 20, 30, 40);
```

This comment applies to multiple places in this file